suite-native/atoms/src/VerticalSeparator.tsx

```diff
@@ -0,0 +1,41 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation, TxKeyPath } from '@suite-native/intl';
+
+import { Box } from './Box';
+import { HStack } from './Stack';
+import { Text } from './Text';
+
+type VerticalSeparatorProps = {
+    parentPadding: number;
+    separatorTitle: TxKeyPath;
+};
+
+const separatorStyle = prepareNativeStyle<{ parentPadding: number }>(
+    (utils, { parentPadding }) => ({
+        borderColor: utils.colors.borderElevation0,
+        borderWidth: utils.borders.widths.small,
+        flex: 1,
+        // We want the separator to be full width, but we need to offset it by the parent padding
+        marginHorizontal: -parentPadding,
+    }),
+);
+
+const separatorTitleStyle = prepareNativeStyle(_ => ({
+    paddingHorizontal: 12,
+}));
+
+export const VerticalSeparator = ({ separatorTitle, parentPadding }: VerticalSeparatorProps) => {
```

Comment:
```suggestion
export const TitledDivider = ({ separatorTitle, parentPadding }: VerticalSeparatorProps) => {
```

We already have Divider in atoms, lets follow the naming or rename atoms/Divider to Separator or make it one component with optional title

Reply:
Hmm, we actually already have TextDivider too 🙈 My bad, I will ping Sheny and discuss how we can unify this...

Reply:
Here is TextDivider refactored so it's used in AccountsImport (where it was before) and passphrase form as well. 
https://github.com/trezor/trezor-suite/pull/12333/commits/18d26077c560a2553efa39ab29ead9aae3a02f61